test/mokernels/independent.jl

```diff
@@ -14,14 +15,16 @@
     @test k isa Kernel
     @test k.kernel isa Kernel
 
-    @test kernelmatrix(k, x, y) == kernelmatrix(k, collect(x), collect(y))
+    @test isapprox(
+        kernelmatrix(k, x, y), kernelmatrix(k, collect(x), collect(y)); rtol=1e-6
+    )
```

Comment:
Is the non-standard rtol needed, in particular if we fix the seeds with StableRNGs? Maybe
```suggestion
    @test kernelmatrix(k, x, y) ≈ kernelmatrix(k, collect(x), collect(y))
```
would be sufficient?